src/Controls/src/Core/MergedStyle.cs

```diff
@@ -101,13 +101,18 @@ public void Apply(BindableObject bindable, SetterSpecificity specificity)
 		void Apply(BindableObject bindable)
 		{
 			//NOTE specificity could be more fine grained (using distance)
-			ImplicitStyle?.Apply(bindable, new SetterSpecificity(SetterSpecificity.StyleImplicit, 0, 0, 0));
+			ImplicitStyle?.Apply(bindable, SetterSpecificity.ImplicitSetter);
```

Comment:
So what is interesting, in your numbers on the PR description, before & after both say Allocated: 28.25 KB.

`SetterSpecificity` is a `struct`, and so maybe that `Allocated` column is heap allocations only?

The fact the time improved, does indicate your change helps, though.